apps/client/src/components/Policy/BulletItem/index.tsx

```diff
@@ -0,0 +1,19 @@
+/** @jsxImportSource @emotion/react */
+
+'use client';
+
+import * as S from './style';
+
+interface BulletItemProps {
+  children: React.ReactNode;
+  isDark?: boolean;
+}
+
+const BulletItem: React.FC<BulletItemProps> = ({ children, isDark }) => (
```

Comment:
```suggestion
const BulletItem: React.FC<BulletItemProps> = ({ children, isDark = false }) => (
```

isDark ?? false로 값을 지정하는 것 보단, props에 isDark 값이 없을 때 기본 값을 false로 지정해주는게 좋아보이네요.